### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by antoaravinth**
Added an link to Angular 2 starter links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Table of contents:
 * [Built With Angular 2](http://builtwithangular2.com/)
 * [`#ng-2 Slack Channel`](https://angularbuddies.slack.com/messages/ng-2/) on AngularBuddies ([Sign up](http://www.angularbuddies.com/))
 * [`#angular2 Slack Channel`](https://dartlang.slack.com/messages/angular2) on dartlang.slack.com ([Sign up](https://dartlang-slack.herokuapp.com/))
-
+s
```
Comment (rolandjitsu) [inline README.md]: Is this a typo?

Comment (antoaravinth) [inline README.md]: Oh sorry it's an typo! I guess I did cmd + s thinking I was writing in a file. I created a PR via web. Sorry for it!!

Comment (rolandjitsu) [inline README.md]: No worries, just make sure to change it and we'll merge it.

**Comment by rolandjitsu** [comment]
@antoaravinth please make another pull request if you still want to add the resource.

**Pull request closed without merging**